src/logic/calculate.js

```diff
@@ -0,0 +1,37 @@
+import operate from './operate';
+
+export default function calculate(calcData, buttonName) {
+  let { total, next, operation } = calcData;
+  let result;
+  switch (buttonName) {
+    case '+':
+    case '-':
+    case 'X':
+    case '÷':
```

Comment:
- Implement operations for `+/-` and `.` buttons

Reply:
Done